CLAUDE.md

```
@@ -0,0 +1,128 @@
+
+| VID:PID | 设备 | 用途 |
+|---------|------|------|
+| 0483:2000 | WUJIHAND (sboard) | 灵巧手关节控制 |
+| 0483:5700 | WujiHand G-Board | 触觉传感器 |
+| 0483:5701 | WujiHand TBoard Bootloader | 触觉板 OTA 模式 |
+
+`wujihand_full.launch.py` 通过扫描 `/sys/bus/usb/devices` 自动按 PID 区分设备，不需要手动填 serial number。
+
+## Topic 命名
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)